### ISSUE TITLE
Bug: Core crashes with mod in use - Kill log pets

### ISSUE THREAD

**Issue by Nelnamara** [azerothcore/mod-detailed-logging#12]
### Current Behaviour

Core crashes after a few minutes due to inability to log player pet kills (Hunter/warlock pet)

### Expected Behaviour

Should be logging the kills of said player pet

### Steps to reproduce the problem

1. Run Core
2. Use Hunter/warlock to kill a creature

### Extra Notes

I commented out the section to further test the mod.  this is the only part the core crashes with.

### AC rev. hash/commit

*

### Operating system

Debian 12

### Custom changes or Modules

Eluna
Worgoblin
guildhouse
playerbots

**Comment by AldebaraanMKII** [comment]
Yeah just got a crash when a pet kills something. The crashlog points to this module.
